app/tasks/maintenance/move_image_urls_task.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Maintenance
+  class MoveImageUrlsTask < MaintenanceTasks::Task
+    def collection
+      Podcast.all
```

Comment:
It can create memory exception? I don't know how much data is stored it would be nice to check it.

Reply:
number of podcast is low. it cannot be a memory problem.